reflex/style.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, Literal, Tuple, Type
 
 from reflex import constants
+from reflex.components.core.breakpoints import Breakpoints, breakpoints_values
```

Comment:
I'm wondering if we can define the breakpoint values within the `Style` class itself (rather than having it be a global). That way users can set it in their global style dict:

```python
app = rx.App(style={"breakpoints": [...]})
```

Since the `convert` method is only used in the Style class, it can pass its breakpoints to it then.

Reply:
@picklelo  I'm trying to implement this, but it seems that `style=` is processed after processing the `rx.breakpoints`. This means the modification happens after it already generates the CSS for the element.